modules/services/cloud-bench/trust_relationship/variables.tf

```diff
@@ -9,3 +9,9 @@ variable "role_name" {
   description = "The name of the Service Account that will be created."
   default     = "sysdigcloudbench"
 }
+
+variable "reuse_workload_identity_pool" {
+  type        = bool
+  description = "Reuse existing workload identity pool"
```

Comment:
i would enhance description a bit , like
> reuse existing workload identity pool from previous deployments (sysdigcloud) , to [overcome current-redeploying error](https://github.com/sysdiglabs/terraform-google-secure-for-cloud#q-getting-error-creating-workloadidentitypool-googleapi-error-409-requested-entity-already-exists)

Reply:
Is it allowed to use rich text or markdown in the variables description? I don't think so. I extended the description a bit, but I still think this will be mostly for internal usage only. If we want to make it 100% right I would also update the README.md explaining the alternative, and how you still need to undelete the workload identity pool and the provider, then enable the new "reuse_workload_identity_pool" option. But it saves you from importing into the state, which is a big problem when using Terraform Cloud.

Reply:
yep, can add html syntax :D